src/ext/ddtrace.c

```diff
@@ -157,9 +157,16 @@ static PHP_RINIT_FUNCTION(ddtrace) {
     ddtrace_init_span_stacks(TSRMLS_C);
     ddtrace_coms_on_pid_change();
 
-    if (DDTRACE_G(request_init_hook)) {
-        DD_PRINTF("%s", DDTRACE_G(request_init_hook));
+    if (DDTRACE_G(request_init_hook) && access(DDTRACE_G(request_init_hook), F_OK) == 0) {
+        DD_PRINTF("Loading request init hook: %s", DDTRACE_G(request_init_hook));
```

Comment:
I think it would be useful to differentiate the two error messages a little bit more.

E.g.:
- when request init hook is specified: `Loading request init hook from user-defined location: %s`
- when request init hook is not specified: `Loading request init hook from computed default location: %s`